app/components/curriculum-inventory-sequence-block-session-manager.js

```diff
@@ -82,7 +116,8 @@ export default Component.extend({
 
   saveChanges: task(function * () {
     let sessions = this.get('linkedSessionsBuffer');
-    yield this.sendAction('save', sessions);
+    let excludedSessions = this.get('excludedSessionsBuffer');
+    yield this.sendAction('save', sessions, excludedSessions);
```

Comment:
Linter will start flagging `sendAction` very soon as an old pattern and possibly a bad idea in the future. Instead the action should be passed in like `{{curriculum-inventory-sequence-block-session-manager save=(action 'save')}}` which makes it a function, then `this.sendAction('save', sessions, excludedSessions);` can be replaced with `this.get('save')(sessions, excludedSessions);`